Create bug bounty program page in Lisk docs - Closes #1506 (#1597)
LiskHQ/lisk-docs: docs/modules/ROOT/pages/intro/bug-bounty-program.adoc

= Lisk Bug Bounty Program
Chris Braithwaite <christopher.braithwaite@lightcurve.io> Mona Bärenfänger <mona@lightcurve.io>
:description: This page covers the submission guidelines regarding how to receive remuneration for locating and reporting security vulnerabilities and bugs in the Lisk core.
:toc: preamble
:idprefix:
:idseparator: -
:imagesdir: ../../assets/images

:url_submission_form: https://lisk.com/contact/ive-found-bug/form

> image::intro/bug-bounty-update.jpeg[align="center"]

Reviewer: The quality of the images gets compromised when zoomed in. Please use the images from the following URLs:

- https://lisk.com/sites/default/files/media/bugbounty-hero_6.svg
- https://lisk.com/sites/default/files/pictures/2020-03/Bugbounty-payouttable-v2.svg
- https://lisk.com/sites/default/files/pictures/2022-06/Group%201062.png

Author: Cool..Many thanks for that Tahla. I will replace them 👍

== Overview

> The Lisk bug bounty program is a long-standing initiative designed to offer an incentive in the form of a remuneration payment to anyone who locates any bugs or vulnerabilities that may exist within the existing Lisk core network.

Reviewer: ```suggestion
The Lisk bug bounty program is a long-standing initiative designed to offer an incentive in the form of a remuneration to anyone who locates any bugs or vulnerabilities that may exist within the existing Lisk core network.
```

Author: Thanks for the feedback. If you wish to remove the word  'payment' then we would also need to remove the indefinite article 'a' as well, as this is then redundant when used with an uncountable noun such as 'remuneration'
I  agree and have updated it accordingly 👍

> = Bug bounty program

Reviewer 2: ```suggestion
= Lisk Bug Bounty Program
```

Chris Braithwaite <christopher.braithwaite@lightcurve.io> Mona Bärenfänger <mona@lightcurve.io>
:description: This page covers the submission guidelines regarding how to receive remuneration for locating and reporting security vulnerabilities and bugs in the Lisk core.
:toc: preamble
:idprefix:
:idseparator: -
:imagesdir: ../../assets/images

:url_submission_form: https://lisk.com/contact/ive-found-bug/form

> image:intro/bug-bounty-update.jpeg[align="center"]

Reviewer 2: ```suggestion
image::intro/bug-bounty-update.jpeg[align="center"]
```

use block image syntax here

Firstly, please ensure before making any submissions that you have reviewed our bug bounty program policy rules below.

== Guidelines

=== Focus on Lisk Core

Only vulnerabilities and bugs in Lisk Core are being considered.
Focus on the master branch and, if one exists, the latest Betanet branch only.
Right now any vulnerability existing within Lisk Core is likely to be present in the Lisk SDK as well.

=== Stay on your private network

Only perform all testing on your own private network.
A report is only eligible if it is clear that the reporter did not abuse this bug on one of the public networks.

=== Keep it to yourself

It is forbidden to disclose reported bugs to the public.
Doing so will forfeit any potential remuneration.

=== Validity of vulnerabilities

Vulnerabilities that were already submitted, are already known to us, or are fixed by the implementation of an existing LIP are not eligible for any remuneration.
Serial vulnerabilities caused by the same underlying issue are treated as a single vulnerability.

=== Remuneration

The following table illustrated below lists the current remuneration structure for identifying bugs and vulnerabilities.

> image:intro/remuneration1.png[ align="center"]

Reviewer 2: ```suggestion
image::intro/remuneration1.png[ align="center"]
```

=== Report your Bug

> To file a report, use this link to access the {url_submission_form}[submission form^].
> Alternatively, if you prefer to file a report via email, use the following address: security@lisk.com
> 
> image:intro/bug-report-form.png[align="center" ,400]

Reviewer 2: ```suggestion
image:intro/bug-report-form.png[role="related thumb right",200]
To file a report, use this link to access the {url_submission_form}[submission form^].
Alternatively, if you prefer to file a report via email, use the following address: security@lisk.com
```
to improve image/text alignment
